apollo-federation/src/subgraph/spec.rs

```diff
@@ -320,6 +321,15 @@ impl FederationSpecDefinitions {
         }
     }
 
+    /// scalar ContextFieldValue
+    pub fn contextfieldvalue_scalar_definition(&self, alias: &Option<Name>) -> ScalarType {
```

Comment:
probably also a typo?
```suggestion
    pub fn context_field_value_scalar_definition(&self, alias: &Option<Name>) -> ScalarType {
```

Reply:
This case is [similar to above](https://github.com/apollographql/router/pull/6380#discussion_r1868170997), where we're following an existing convention (e.g. `fieldset_scalar_definition()` for `FieldSet`). Definitely down to change the convention across the crate in a follow-up.